test/binCases/no-options/stdin.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+module.exports = function testAssertions(code, stdout, stderr) {
+	expect(code).toBe(2);
+	expect(stdout).toEqual(expect.anything());
+	expect(stdout[0]).toContain("Hash: ");
+	expect(stdout[1]).toContain("Version: ");
+	expect(stdout[2]).toContain("Time: ");
+	expect(stdout[4]).toContain("");
+	expect(stdout[5]).toContain("WARNING");
+	expect(stdout[6]).toContain("The \'mode\' option has not been set");
```

Comment:
The message should be `The 'mode' option has not been set. Set 'mode' option to 'development' or 'production' to enable defaults for this environment.`

I know it's passing but it would be better to show the whole message

Reply:
I had initially thought to keep the whole message but then I read the [blog post](https://medium.com/webpack/webpack-4-released-today-6cdb994702d4) that `production` is default `mode` so we will change the latter part of this message.